portfolio-project-react/src/components/WorkExpItem/WorkExpItem.jsx

```diff
@@ -0,0 +1,161 @@
+    setEditExp(false);
+    setCompNameInput(data.compName);
+    setStartDateInput(data.startDate);
+    setEndDateInput(data.endDate);
+    setIsCurrentEmpInput(data.isCurrentEmp);
+    setJobDescriptionInput(data.jobDescription);
+  }, [isEdit]);
+  return (
+    <div className={styles.expItemContainer}>
+      <div className={styles.topSection}>
+        {editExp ? (
+          <div>
+            <label className={styles.companyNameLabel}>Company Name: </label>
+            <input
+              className={styles.compNameInput}
+              value={compNameInput}
+              onChange={(e) => {
+                setCompNameInput(e.target.value);
+              }}
+            />
+          </div>
+        ) : (
+          <h4 className={styles.companyName}>
+            {index + 1 + ". " + data.compName}
+          </h4>
+        )}
+
+        {isEdit && !editExp && (
+          <div className={styles.headerEvents}>
+            <button
+              className={styles.deleteBtn}
+              onClick={(e) => {
+                onDelete(index);
+              }}
+            >
+              <img src={deleteIcon} />
+            </button>
+            <button
+              className={styles.editBtn}
+              onClick={(e) => {
+                setEditExp(true);
+              }}
+            >
+              <img src={editIcon} />
+            </button>
+          </div>
+        )}
+      </div>
+
+      {editExp ? (
+        <div className={styles.dateSection}>
+          <div className={styles.dateSectionLabels}>
+            <label>{LABEL_START_DATE}</label>
+            <label>{LABEL_END_DATE}</label>
+            <label></label>
+          </div>
+          <div className={styles.dateSectionValues}>
+            <input
+              type="date"
+              required
+              max={moment(new Date()).toISOString().split("T")[0]}
+              value={startDateInput}
+              onChange={(e) => setStartDateInput(e.target.value)}
+            />
+            <input
+              type="date"
+              required
+              max={moment(new Date()).toISOString().split("T")[0]}
+              value={endDateInput}
+              disabled={isCurrentEmpInput}
+              onChange={(e) => setEndDateInput(e.target.value)}
+            />
+            <div className={styles.checkboxSection}>
+              <input
+                type="checkbox"
+                className={styles.checkbox}
+                checked={isCurrentEmpInput}
+                onChange={(e) => {
+                  setIsCurrentEmpInput(e.target.checked);
+                  setEndDateInput("");
+                }}
+              />
+              <label className={styles.labelEmployer}>{LABEL_CURR_EMP}</label>
+            </div>
+          </div>
+        </div>
+      ) : (
+        <h5 className={styles.jobDuration}>
+          {data.startDate +
+            " - " +
+            (data.isCurrentEmp ? "Present" : data.endDate)}
```

Comment:
[suggestion] to keep our markup clean, we can extract such logics above into a variable and simply plug the conditional value + concatenated string stored in that variable here.